### PR TITLE
[TRANSLATION][RUNONCE] Add Bulgarian translation.

### PR DESCRIPTION
**Pull request by WindowxDeveloper**
## Purpose

Add Bulgarian translation for RUNONCE.

### CODE REVIEW

**Comment by Fraizeraust** [comment]
It's recommended that you use a text editor for your translations and a Git client for committing instead of using the Git text editor. 

Furthermore, each contributor must sign their contributions with a real name and real E-mail. You may want to [check this](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations) for further information.

**Comment by HBelusca** [comment]
Please also check that the new resource files are encoded in UTF-8 **WITHOUT BOM**.

**Comment by katahiromz** [comment]
https://reactos.org/wiki/Translation_Introduction

**Comment by WindowxDeveloper** [comment]
Thanks for fixing this mistake! :) I'm just new to Github, but as soon as I get into programming, I would be better

File: base/system/runonce/lang/bg-BG.rc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ LANGUAGE LANG_BULGARIAN, SUBLANG_DEFAULT
```
[the remark this answers is not in the document]
Comment (WindowxDeveloper) [reply base/system/runonce/lang/bg-BG.rc]: Thanks for fix.

**Pull request merged**